bytes/README.md

```diff
@@ -132,21 +132,68 @@ VWS_PUBLIC_KEY_B64=""
 
 ## Design
 
-The overall view of the application is as follows.
+We now include two levels of design, according to the [C4 model](https://c4model.com/).
+
+
+### Design: C2 Container level
+The overall view of the code is as follows.
+
+```mermaid
+graph
+    User((User))
+    Rocky["Rocky<br/><i>Django App</i>"]
+    Bytes{"Bytes<br/><i>FastAPI App"}
+    RabbitMQ[["RabbitMQ<br/><i>Message Broker"]]
+    Scheduler["Scheduler<br/><i>Software System"]
+    Boefjes["Boefjes<br/><i>Python App"]
+
+
+    Boefjes -- GET/POST Raw/Meta --> Bytes
+    User -- Interacts with --> Rocky
+    Rocky -- GET/POST Raw/Meta --> Bytes
+
+    Bytes -- "publish(RawFileReceived)" --> RabbitMQ
+    Scheduler --"subscribe(RawFileReceived)"--> RabbitMQ
+    Scheduler --"GET BoefjeMeta"--> Bytes
+```
+
+### Design: C3 Component level
+The overall view of the code is as follows.
 
 ```mermaid
 graph LR
-    U[User] -- BoefjeMeta --> A1[API] -- save BoefjeMeta --> B1[Meta Repository]
-    B1[Meta Repository] -- BoefjeMeta --> RDB
-    B[Meta Repository] -- hash RawFile --> HR[Hasher] -- Hash --> B[Meta Repository]
-    U[User] -- RawFile --> A2[API] -- save RawFile --> B[Meta Repository] -- save Hash --> H[Hash Repository]
-    H[Hash Repository] -- Hash --> T[Third Party]
-    B[Meta Repository] -- save RawFile --> R[Raw Repository] -- RawFile --> E[EnriptionMiddleware]
-    E[EnriptionMiddleware] -- Encrypted Data --> Disk -- Encrypted Data  --> E[EnriptionMiddleware]
+    User -- BoefjeMeta --> APIR1
+    User -- NormalizerMeta --> APIR2
+    User -- RawFile --> APIR3
+
+
+    User[User]
+
+    APIR1 -- save BoefjeMeta --> MR
+    APIR2 -- save NormalizerMeta --> MR
+    APIR3 -- save RawFile --> MR
+
+    subgraph API["Bytes API"]
+        APIR1[API Route]
+        APIR2[API Route]
+        APIR3[API Route]
+    end
+
+    subgraph Bytes["Bytes Domain"]
+        APIR3 -- "publish(RawFileReceived)" --> EM[EventManager]
+        MR[Meta Repository] -- Raw  --> H[Hasher] -- Hash --> MR[Meta Repository]
+        MR[Meta Repository] -- save Hash --> HR[Hash Repository]
+        MR[Meta Repository] -- save RawFile --> R[Raw Repository]
+        R[Raw Repository] -- RawFile --> E[EnriptionMiddleware]
```

Comment:
```suggestion
        R[Raw Repository] -- RawFile --> E[EncryptionMiddleware]
```

Probably a copy-paste mistake; there are more errors for this one in the document

Reply:
Good spot!